content/articles/2019-05-forcats-0-4-0.Rmarkdown

```diff
@@ -0,0 +1,87 @@
+---
+title: forcats 0.4.0
+author: Mara Averick
+date: '2019-05-10'
+slug: forcats-0-4-0
+description: > 
+  What's new in forcats 0.4.0?
+categories:
+  - package
+photo:
+  url: https://www.pexels.com/photo/close-up-photo-of-a-hand-holding-three-white-kittens-1643456/
```

Comment:
🚨🚨🚨
There are only _three_ cats in this photo!!!
🚨🚨🚨

Reply:
I know! 😂

Reply:
The fourth cat is hiding behind the other 3